tuxemon/core/event/actions/variable_math.py

```diff
@@ -40,7 +40,7 @@ class VariableMathAction(EventAction):
     valid_parameters = [
         (str, "var"),
         (str, "operation"),
-        (float, "value")
+        (str, "value")
```

Comment:
Needed because this is passed through the `number_or_variable` function, which expects a string rather than a float.